server/datastore/mysql/labels.go

```diff
@@ -15,6 +15,10 @@ import (
 )
 
 func (ds *Datastore) ApplyLabelSpecs(ctx context.Context, specs []*fleet.LabelSpec) (err error) {
+	return ds.ApplyLabelSpecsWithAuthor(ctx, specs, 0)
```

Comment:
Should the argument be `authorID *uint` and the default `nil`?

Reply:
🤔 Good question -- there's pros and cons to both.  Since this works with specs, we'll never have an actual authorId pointer to use, so we'd always have to cast an int to a pointer just to use it as an arg here.  On the other hand, `nil` is a nicer arg here than `0` since we'll never actually put `0` in the database, and simplifies the SQL.  I like that, I'll make the update.